docs/examples/example_survival.ipynb

```diff
@@ -0,0 +1,196 @@
+    "\n",
+    "        if self.booster_params is None:\n",
+    "            booster_params = common_params\n",
+    "        else:\n",
+    "            booster_params = common_params | self.booster_params\n",
+    "\n",
+    "        self.bst_ = xgb.train(booster_params, dtrain, num_boost_round=self.n_estimators)\n",
+    "        return self\n",
+    "\n",
+    "    def predict(self, X):\n",
+    "        # Check if fit has been called\n",
+    "        check_is_fitted(self)\n",
+    "        # X contains \"censored\" column\n",
+    "        features = set(X.columns) - {\"censored\"}\n",
+    "        if set(np.unique(X[\"censored\"])) != {0, 1}:\n",
+    "            raise ValueError(\"censored column should be binary.\")\n",
+    "        dtest = xgb.DMatrix(X[list(features)], enable_categorical=True)\n",
+    "        return self.bst_.predict(dtest)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fitting the MetaLearner\n",
+    "\n",
+    "Finally we can instantiate any MetaLearner with the ``SurvivalModel`` as the nuisance model and estimate the CATEs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lightgbm import LGBMClassifier, LGBMRegressor\n",
+    "from metalearners import TLearner, DRLearner, RLearner\n",
+    "from metalearners.utils import simplify_output\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "tlearner = TLearner(False, 2, SurvivalModel)\n",
+    "tlearner.fit(X, y, w)\n",
+    "cate_estimates_tlearner = tlearner.predict(X, True)\n",
+    "\n",
+    "drlearner = DRLearner(\n",
+    "    False,\n",
+    "    2,\n",
+    "    nuisance_model_factory=SurvivalModel,\n",
+    "    propensity_model_factory=LGBMClassifier,\n",
+    "    treatment_model_factory=LGBMRegressor,\n",
+    "    propensity_model_params={\"verbose\": -1},\n",
+    "    treatment_model_params={\"verbose\": -1},\n",
+    ")\n",
+    "drlearner.fit(X, y, w)\n",
+    "cate_estimates_drlearner = drlearner.predict(X, True)\n",
+    "\n",
+    "rlearner = RLearner(\n",
+    "    False,\n",
+    "    2,\n",
+    "    nuisance_model_factory=SurvivalModel,\n",
+    "    propensity_model_factory=LGBMClassifier,\n",
+    "    treatment_model_factory=LGBMRegressor,\n",
+    "    propensity_model_params={\"verbose\": -1},\n",
+    "    treatment_model_params={\"verbose\": -1},\n",
+    ")\n",
+    "rlearner.fit(X, y, w)\n",
+    "cate_estimates_rlearner = rlearner.predict(X, True)\n",
+    "\n",
+    "plt.hist(simplify_output(cate_estimates_tlearner), bins=20, alpha=0.5, label=\"TLearner\")\n",
+    "plt.hist(\n",
+    "    simplify_output(cate_estimates_drlearner), bins=20, alpha=0.5, label=\"DRLearner\"\n",
+    ")\n",
+    "plt.hist(simplify_output(cate_estimates_rlearner), bins=20, alpha=0.5, label=\"RLearner\")\n",
+    "plt.legend()\n",
+    "plt.show()"
```

Comment:
It might be neat to add an x-axis label, including aunit, and a plot title, e.g. Histogram of CATE estimates, to the plot.

Reply:
[8e16c25](https://github.com/Quantco/metalearners/pull/38/commits/8e16c25fd392834af347324d3e7610f5d0b2a755)